plugins/git/README.md

```diff
@@ -42,12 +42,12 @@ plugins=(... git)
 | gcans!               | git commit -v -a -s --no-edit --amend                                                                                            |
 | gcam                 | git commit -a -m                                                                                                                 |
 | gcas                 | git commit -a -s                                                                                                                 |
-| gcasm                | git commit -a -s -m                                                                                                                  |
+| gcasm                | git commit -a -s -m                                                                                                              |
 | gcsm                 | git commit -s -m                                                                                                                 |
 | gcb                  | git checkout -b                                                                                                                  |
 | gcf                  | git config --list                                                                                                                |
 | gcl                  | git clone --recurse-submodules                                                                                                   |
-| gccd                 | git clone --recurse-submodules "$@" && cd "$(basename $_ .git)"                                                                  |
+| gccd                 | git clone --recurse-submodules "<span>$</span>@" && cd "<span>$</span>(basename \<span>$</span>_ .git)"                          |
```

Comment:
I think that final backslash should not be there! Besides that, LGTM!

Reply:
Missed removing that when I figured out how to escape the $. Fixed!